apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -12,48 +12,28 @@ import {
   searchAssets,
   filterOutBackgrounds
 } from '@cdo/apps/code-studio/assets/searchAssets';
+import experiments from '@cdo/apps/util/experiments';
+import Button from '@cdo/apps/templates/Button';
 
 const MAX_SEARCH_RESULTS = 40;
 const MAX_HEIGHT = 460;
 
-const animationPickerStyles = {
-  allAnimations: {
-    color: color.purple,
-    fontFamily: "'Gotham 7r', sans-serif",
-    cursor: 'pointer'
-  },
-  breadCrumbs: {
-    margin: '8px 0',
-    fontSize: 14,
-    display: 'inline-block'
-  },
-  pagination: {
-    float: 'right',
-    display: 'inline',
-    marginTop: 10
-  },
-  emptyResults: {
-    paddingBottom: 10
-  },
-  navigation: {
-    minHeight: 30
-  }
-};
-
 export default class AnimationPickerBody extends React.Component {
   static propTypes = {
     is13Plus: PropTypes.bool,
     onDrawYourOwnClick: PropTypes.func.isRequired,
     onPickLibraryAnimation: PropTypes.func.isRequired,
     onUploadClick: PropTypes.func.isRequired,
+    onAnimationSelectionDone: PropTypes.func.isRequired,
     playAnimations: PropTypes.bool.isRequired,
     libraryManifest: PropTypes.object.isRequired,
     hideUploadOption: PropTypes.bool.isRequired,
     hideAnimationNames: PropTypes.bool.isRequired,
     navigable: PropTypes.bool.isRequired,
     defaultQuery: PropTypes.object,
     hideBackgrounds: PropTypes.bool.isRequired,
-    canDraw: PropTypes.bool.isRequired
+    canDraw: PropTypes.bool.isRequired,
+    selectedAnimations: PropTypes.arrayOf(PropTypes.object).isRequired
```

Comment:
we should also specify the shape of the selectedAnimation object here, so we should pull this shape into a common place to be shared across components